File: rafael/controle-de-fluxo/ComparaValores.java
class ComparaValores {
	public static void main(String[] args) {
		double primeiro = Math.random();
		double segundo = Math.random();
	
		System.out.println("Primeiro: " + primeiro);
		System.out.println("Segundo: " + segundo);

		if(primeiro > segundo) {
			System.out.println("Primeiro > Segundo");
		} else if(primeiro < segundo) {
			System.out.println("Segundo > Primeiro");
		} else {
			System.out.println("Primeiro = Segundo");
		}
	}
}

File: rafael/controle-de-fluxo/ComparaValores.cs
class ComparaValores
{
	static void Main()
	{
		System.Random gerador = new System.Random();

		double primeiro = gerador.NextDouble();
		double segundo = gerador.NextDouble();
	
		System.Console.WriteLine("Primeiro: " + primeiro);
		System.Console.WriteLine("Segundo: " + segundo);

		if(primeiro > segundo)
		{
			System.Console.WriteLine("Primeiro > Segundo");
		} 
		else if(primeiro < segundo)
		{
			System.Console.WriteLine("Segundo > Primeiro");
		} 
		else
		{
			System.Console.WriteLine("Primeiro = Segundo");
		}
	}
}
